content/components/treeview.mdx

```diff
@@ -0,0 +1,579 @@
+
+## Nesting
+
+Nodes are visually nested using whitespace and alignment.
+
+### Align leading icons
+
+<Box as="figure" display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} m={0} sx={{gap: 3}}>
+  <Box flex={1}>
+    Icons at the same level are center-aligned. The chevron should not break the alignment: parent nodes and child nodes
+    keep their leading visuals aligned.
+  </Box>
+  <img
+    width="456"
+    role="presentation"
+    alt="19 leadingVizAlignment"
+    src="https://user-images.githubusercontent.com/2313998/187768704-08fac391-f331-4a47-9181-073f1360677c.png"
+  />
+</Box>
+
+### Indentation
+
+<Box as="figure" display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} m={0} sx={{gap: 3}}>
+  <Box flex={1}>Each level of nesting increments the size of the indentation.</Box>
+  <img
+    width="456"
+    role="presentation"
+    alt="20 nodesWithIndentations"
+    src="https://user-images.githubusercontent.com/2313998/187768706-1910025a-f69b-487f-987b-319f3022cd09.png"
+  />
+</Box>
+
+### Nesting level indicator lines
+
+In devices with `:hover` support, the nesting indicator lines fade in when the user mouses over the entire component, or when there's focus inside the component.
+
+On devices without `:hover`, the nesting indicator lines appear at all times.
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <img
+    width="456"
+    role="presentation"
+    alt="21 1 showLinesOnHover"
+    src="https://user-images.githubusercontent.com/2313998/187768708-805527f9-3b7c-4f6b-ae37-e232f83916b6.png"
+  />
+  <img
+    width="456"
+    role="presentation"
+    alt="21 2 alwaysShowLinesOnTouch"
+    src="https://user-images.githubusercontent.com/2313998/187768711-2702a7aa-e902-4568-ab99-4570e28bf646.png"
+  />
+</Box>
+
+### Handling deeply nested nodes
+
+Navigating through deeply nested nodes can be cumbersome and visually clunky. If accessing nodes deeper than 10 levels deep is a common interaction for your use-case, reconsider whether a tree view is the best pattern.
+
+Tree views are designed to be horizontally compact in order to support many levels of nesting and long node labels. This makes content truncation and horizontal overflow less likely to occur for deeply nested nodes, but infinite nesting makes those cases impossible to prevent entirely.
+
+There are some strategies to avoid a tree view that horizontally overflows it's container:
+
+- Allow the tree view's container to be manually resized
+- Once a certain depth is reached, start the tree view from a deeper parent node and provide a way to navigate back up the tree
+- Remove leading and trailing visuals
+
+If a tree view is truly the best pattern for your use case and none of the suggested strategies prevent horizontal overflow, opt for horizontal scrolling as a last resort instead of hiding overflowing content.
```

Comment:
```suggestion
If a tree view is truly the best pattern for your use-case and none of the suggested strategies prevent horizontal overflow, opt for horizontal scrolling as a last resort instead of hiding overflowing content.
```

Reply:
You used the hyphenated version above already. So we should probably stick to one.